### DISCUSSION TITLE
AdGuardHome Docker Container needs to be restarted after every boot-up, otherwise it doesn't respond.

### DISCUSSION THREAD

**Discussion by flavorgold1** [AdguardTeam/AdGuardHome#6045] [General]
Before I open an issue, I just want to check if this is something only I'm experiencing or something common.

I'm on the latest Ubuntu LTS 22.04.

I run AdGuardHome via Docker.

And, all was well until a week ago.

Since then, whenever I restart the machine, I've needed to restart the AdGuardHome docker container as well, just once.

Otherwise, it doesn't seem to work/respond to queries on port 53.

I can't see anything in the logs that could point at what the underlying issue could be.

**Comment by antoniocifu** [comment]
The same thing happens to me. 
Every time I restart my server, Adguardhome running on docker stops and I have to restart it manually.
Likewise nothing appears in the logs

**Reply by mmedni** [reply]
Hi!
 i had the same problem running Adguard home in Openmedivault Docker Compose, it wouldn't automatically start after server restart.
 I also noticed that Adguard time was different than my server time. I took a wild guess, that this could be the cause for the issue and mounted additional volume for Adguard home container:
 - /etc/localtime:/etc/localtime:ro
 
 And for me it solved the issue.

**Reply by antoniocifu** [reply]
Thanks for your reply. It's true that it worked to prevent AdGuardHome from crashing on reboots. 

However, I'm still having problems with other containers that depend on it and keep breaking, but that's a different issue I'll have to deal with Docker. Thanks again!

**Comment by mike391** [comment]
Me as well no logs. I have a Cron job to restart the container 5 mins after bootup as a workaround.
